### DISCUSSION TITLE
Updating Livewire to v3.5.0 breaks Alpine/Livewire functionality when using wire:poll requests.

### DISCUSSION THREAD

**Discussion by andrasbacsai** [livewire/livewire#8514] [7. Bugs (archived)]
### Livewire version

v3.5.0

### Laravel version

v10.48.12

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

TLDR: In version 3.5.0, a `data-has-alpine-state` attribute is added to an Alpine/Livewire components, which causes functionality to break.

- This [code](https://github.com/coollabsio/coolify/tree/f5eaedfc728144735c4e6039c66a3d9454d77444/resources/views/livewire/dashboard.blade.php#L124-L128) works in v3.4.9 but not in v3.5.0. The `cleanup_queue` wire action does not work after the first `wire:poll=get_deploments` update occurs.

- The Alpine/Livewire modal component can be found [here](https://github.com/coollabsio/coolify/tree/f5eaedfc728144735c4e6039c66a3d9454d77444/resources/views/components/modal-confirmation.blade.php). It uses Alpine's `x-teleport` directive to add the modal to the `body` element.

- Here is a short video: https://github.com/livewire/livewire/assets/5845193/0c295034-1412-406f-93d7-97134447abaa

### Repository or Wirebox URL

https://github.com/coollabsio/coolify/tree/f5eaedfc728144735c4e6039c66a3d9454d77444

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.
